docs/src/metadata.md

```diff
@@ -1,155 +1,80 @@
 # Metadata
 
+```@setup metadata-example
+using JLSO, Dates
+JLSO.save("breakfast.jlso", :food => "☕️🥓🍳", :cost => 11.95, :time => Time(9, 0))
+```
+
 Manually reading JLSO files can be helpful when addressing issues deserializing objects or to simply to help with reproducibility.
 
-```@repl metadata-example
-using JLSO
+```@example metadata-example
+using JLSO;
+
 jlso = read("breakfast.jlso", JLSOFile)
 ```
 
 Now we can manually access the serialized objects:
-```@repl metadata-example
+```@example metadata-example
```

Comment:
Switching most instances of `@repl` to `@example` as `strict=true` will only error in the latter case.